archetypes/mule-app/src/main/resources/archetype-resources/pom.xml

```diff
@@ -15,8 +15,8 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<mule.version>${muleVersion}</mule.version>
         <mule.tools.version>@project.version@</mule.tools.version>
-		<jdk.version>1.7</jdk.version>
-		<junit.version>4.9</junit.version>
+		<jdk.version>1.8</jdk.version>

```

Comment:
Mule does only require 1.7 to compile forcing this it may be a little too much right?